.gitignore

```diff
@@ -2,3 +2,4 @@
 .env
 benches.json
 target
+node_modules
```

Comment:
Can we use a subdir as the root? `target` even?

Reply:
How would this help us avoid a `node_modules` bloating the root dir?

Reply:
The `node_modules` in already nested inside `test_browser`; only the `.gitignore` for it is located in the root.